test/non-regression.js

```diff
@@ -229,6 +229,19 @@ describe("verify", () => {
       );
     });
 
+    it("enum declaration", () => {
+      verifyAndAssertMessages(
+        `
+          enum E {
+            A,
+            B,
+          }
+          E.A;
+        `,
+        { "no-unused-vars": 1, "no-undef": 1 }
```

Comment:
Can you also add a test for an unused enum, which should be reported?

Reply:
OK, will add!